src/pages/home.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Bookitems from '../components/bookItem';
+import Addbook from '../components/addBooks';
+
+const BookPage = () => (
+  <main>
+    <Bookitems />
+    <Addbook />
+  </main>
+);
```

Comment:
- [ ] Currently, you are having two forms for adding books and it is because you are calling the Addbook component in here. Kindly remove this one and its import and maintain the one you called inside of the BookItem component as that's exactly where it should be. 👍